Makefile

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/make -f
+PROJECTNAME=$(shell basename "$(PWD)")
+
+## help: Get more info on make commands.
+help: Makefile
+	@echo " Choose a command run in "$(PROJECTNAME)":"
+	@sed -n 's/^##//p' $< | column -t -s ':' |  sed -e 's/^/ /'
+.PHONY: help
+
+## format: Formats only *.go (excluding *.pb.go *pb_test.go). Runs `gofmt & goimports` internally.
+format:
+	find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
+	find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/celestiaorg/celestia-node
+.PHONY: format
+
+## lint: Linting *.go files using golangci-lint. Look for .golangci.yml for the list of linters.
+lint:
+	@echo "--> Running linter"
+	@golangci-lint run
+.PHONY: lint
```

Comment:
Don't think there is a need to keep `format`, as `lint` covers that.

Reply:
OMT: I'm starting to go a little bit overboard, but having a super-duper combo like `make format && make lint && make test` as a one command might be helpful 🧐 wdyt?

Reply:
I think for now its fine as is. We dont have to make it perfect. A developer is free to use which command they want.

Reply:
> While triggering gofmt, lint will only check if the file is formated, but not do the formating itself. It's one of the reason why I make the format as a separate command.

Oh, ok I missed that. Thanks!